script.js

```diff
@@ -1,4 +1,435 @@
+// set global variables
+// to track games and progress
+var numGamesDraw = 0;
+var numGamesWon = 0;
+var numGamesPlayed = 0;
+
+// to set game mode
+var currentGameMode = "enter user name";
+console.log(currentGameMode);
+
+// to set user name
+var userName = "";
+
+// for Korean game
+var lastWinner = 0;
+
+//set function to return percentage win
+var percentageWin = function () {
+  var winRate = Math.floor((numGamesWon / numGamesPlayed) * 100);
+  return winRate;
+};
+
+// function to generate random computer move
+var randomItem = function () {
+  var random = Math.floor(Math.random() * 3);
+  var programMove = "stone";
+  if (random == 1) {
+    programMove = "scissors";
+  }
+  if (random == 2) {
+    programMove = "paper";
+  }
+  return programMove;
+};
```

Comment:
Great job decomposing this into a function on it's own.
I would name the function programMove()
and the variable programMove to randomItem instead.